node/nodeTesting.go

```diff
@@ -215,6 +215,56 @@ func (n *Node) GenerateAndSendBulkTransactionsOneByOne(receiverHex string, value
 	return nil
 }
 
+func (n *Node) SendBulkTransactions(txs []*transaction.Transaction) error {
+	transactionsByShards := make(map[uint32][][]byte, 0)
+
+	for _, tx := range txs {
+		shardId := n.shardCoordinator.ComputeId(state.NewAddress(tx.SndAddr))
```

Comment:
Please make use of the AddressConverter interface instead of creating a new address from byte slice.

Reply:
done. node has an address converter and CreateTransaction method was added to it

Reply:
not done yet

Reply:
done. used address converter